components/layout/Footer.jsx

```diff
@@ -0,0 +1,25 @@
+import Link from "next/link";
+import Anchor from "../ui/Anchor";
+import anchors from "../../data/footerAnchor";
+
+export default function Footer() {
+  return (
+      <footer className="flex mx-10 text-white flex-col md:flex-row  md:justify-between  mb-12 mt-17.25  md:mt-31.75 xl:mt-33.75">
+        <p>
```

Comment:
This text here has some space between which you didn't include.

Yours:
![image](https://user-images.githubusercontent.com/90575886/211770558-7646c72e-d7a0-4f09-87ea-beef9e3668f8.png)

Figma:

![image](https://user-images.githubusercontent.com/90575886/211770630-d5ba97a7-66e8-4295-b785-c37eb9f9b229.png)


There is a gap of 10px, and it should looks exactly like this one below:

![image](https://user-images.githubusercontent.com/90575886/211770764-31ed8c75-04b8-4872-902c-62e9b18fbea8.png)

I think the way you implemented the last one, could work on the above also to have the exact result as on the design

Reply:
Sure, Will do as requested.